packages/coding-standards/shopsys-coding-standard.yml

```diff
@@ -0,0 +1,215 @@
+    ObjectCalisthenics\Sniffs\Files\FunctionLengthSniff:
+        maxLength: 60
+    ObjectCalisthenics\Sniffs\Metrics\PropertyPerClassLimitSniff:
+        maxCount: 30
+
+parameters:
+    exclude_files:
+        - '*tests/ShopBundle/Test/Codeception/_generated/AcceptanceTesterActions.php*'
+    skip:
+        # private properties should not start with "_"
+        PHP_CodeSniffer\Standards\Squiz\Sniffs\NamingConventions\ValidVariableNameSniff.PrivateNoUnderscore: ~
+        # it is not necessary to have blank line after control structure
+        PHP_CodeSniffer\Standards\Squiz\Sniffs\WhiteSpace\ControlStructureSpacingSniff.NoLineAfterClose: ~
+        # allow empty "catch (Exception $exception) { }"
+        PHP_CodeSniffer\Standards\Generic\Sniffs\CodeAnalysis\EmptyStatementSniff.DetectedCatch: ~
+
+        # private properties should not start with "_"
+        PHP_CodeSniffer\Standards\Squiz\Sniffs\NamingConventions\ValidVariableNameSniff.PrivateNoUnderscore: ~
+        # it is not necessary to have blank line after control structure
+        PHP_CodeSniffer\Standards\Squiz\Sniffs\WhiteSpace\ControlStructureSpacingSniff.NoLineAfterClose: ~
+        # allow empty "catch (Exception $exception) { }"
+        PHP_CodeSniffer\Standards\Generic\Sniffs\CodeAnalysis\EmptyStatementSniff.DetectedCatch: ~
+
+        ObjectCalisthenics\Sniffs\Files\FunctionLengthSniff:
+        # skip known issues for framework
+            - packages/framework/src/DataFixtures/Base/MailTemplateDataFixture.php
+            - packages/framework/src/DataFixtures/Demo/CategoryDataFixture.php
+            - packages/framework/src/DataFixtures/Demo/OrderDataFixture.php
+            - packages/framework/src/DataFixtures/DemoMultidomain/CategoryDataFixture.php
+            - packages/framework/src/DataFixtures/DemoMultidomain/MailTemplateDataFixture.php
+            - packages/framework/src/DataFixtures/DemoMultidomain/OrderDataFixture.php
+            - packages/framework/src/Form/Admin/Administrator/AdministratorFormType.php
+            - packages/framework/src/Form/Admin/Advert/AdvertFormType.php
+            - packages/framework/src/Form/Admin/Article/ArticleFormType.php
+            - packages/framework/src/Form/Admin/Category/CategoryFormType.php
+            - packages/framework/src/Form/Admin/Customer/BillingAddressFormType.php
+            - packages/framework/src/Form/Admin/Customer/DeliveryAddressFormType.php
+            - packages/framework/src/Form/Admin/Customer/UserFormType.php
+            - packages/framework/src/Form/Admin/Order/OrderFormType.php
+            - packages/framework/src/Form/Admin/Payment/PaymentFormType.php
+            - packages/framework/src/Form/Admin/Product/ProductEditFormType.php
+            - packages/framework/src/Form/Admin/Product/ProductFormType.php
+            - packages/framework/src/Form/Admin/Slider/SliderItemFormType.php
+            - packages/framework/src/Form/Admin/Transport/TransportFormType.php
```

Comment:
Same trick can be used here, I guess you get the idea. 

I personally use them when 3+ files with simliar name format reports error from the same checker service.

```diff
 parameters:
     skip:
         ObjectCalisthenics\Sniffs\Files\FunctionLengthSniff:
-            - packages/framework/src/Form/Admin/Administrator/AdministratorFormType.php
-            - packages/framework/src/Form/Admin/Advert/AdvertFormType.php
-            - packages/framework/src/Form/Admin/Article/ArticleFormType.php
-            - packages/framework/src/Form/Admin/Category/CategoryFormType.php
-            - packages/framework/src/Form/Admin/Customer/BillingAddressFormType.php
-            - packages/framework/src/Form/Admin/Customer/DeliveryAddressFormType.php
-            - packages/framework/src/Form/Admin/Customer/UserFormType.php
-            - packages/framework/src/Form/Admin/Order/OrderFormType.php
-            - packages/framework/src/Form/Admin/Payment/PaymentFormType.php
-            - packages/framework/src/Form/Admin/Product/ProductEditFormType.php
-            - packages/framework/src/Form/Admin/Product/ProductFormType.php
-            - packages/framework/src/Form/Admin/Slider/SliderItemFormType.php
-            - packages/framework/src/Form/Admin/Transport/TransportFormType.php
+            - "*/packages/framework/src/Form/Admin/*/*FormType.php"
```